anta/runner.py

```diff
@@ -42,8 +47,19 @@ async def main(
             template_params:
                 - vrf: default
 
+        anta.tests.connectivity:
+        - VerifyReachability:
+            result_overwrite:
+              categories:
+                - "Overwritten category 1"
+              description: "Test with overwritten description"
+              custom_field: "Test run by John Doe"
```

Comment:
Maybe we should state it is not used in report yet and it is for futur use only ?